int64 id
float64[3] drvec
float64[3] dtvec
float64 t_neg
float64 t_pos
float64[7] joints_neg
float64[7] joints_pos
